src/runtime/utils/refreshHandler.ts

```diff
@@ -0,0 +1,79 @@
+import type { RefreshHandlerConfig, RefreshHandler } from '../types'
+import { useRuntimeConfig, useAuth, useAuthState } from '#imports'
+
+export const defaultRefreshHandler: RefreshHandler & {
```

Comment:
May be better to define an interface instead:
```ts
interface DefaultRefreshHandler extends RefreshHandler {
  // ...
}

const defaultRefreshHandler: DefaultRefreshHandler = {
```

Reply:
✅